README.md

```diff
@@ -50,13 +50,13 @@ Examples
 
 To acquire a single certificate for both `example.com` and `*.example.com`, waiting 900 seconds for DNS propagation:
 
-    certbot certonly \\
-      --authenticator dns-godaddy \\
-      --dns-godaddy-credentials ~/.secrets/certbot/godaddy.ini \\
-      --dns-godaddy-propagation-seconds 900 \\
+    sudo certbot certonly \
```

Comment:
sudo is only required depending on how you are using it, it should not be included in the documentation. sudo is not necessary if your user owns the credentials and dns-godaddy!-certbot is not installed with sudo. You shouldn't run `sudo pip install` for instance, or it causes you to have to run sudo to run certbot. Only a snap based install would need sudo.